packages/carousel/src/Carousel.js

```diff
@@ -119,15 +115,15 @@ export const Carousel = ({
               type='prev'
               position='bottom'
               setPosition={arrowsPosition}
-              {...buttonColorProps}
+              {...buttonStyles}
             />
             {showDots && <Dots />}
             <ArrowButton
               ml={3}
               type='next'
               position='bottom'
               setPosition={arrowsPosition}
-              {...buttonColorProps}
+              {...buttonStyles}
             />
```

Comment:
We'd probably want to clone the node that's passed in as a prop to define props like next, position, etc. in case they're not set by the consumer.

```
            {props.arrowButton ? React.cloneElement(props.arrowButton, {
              type: 'next',
              position: 'bottom',
              setPosition: arrowsPosition
            }) || <ArrowButton
              ml={3}
              type='next'
              position='bottom'
              setPosition={arrowsPosition}
            />
```

Reply:
I made this suggested change for `position: side` buttons only since those are the ones i'm focused on and incorporating the change for `position: top/bottom` would add more dynamic padding/margin code changes.